components/Card/DeleteConversationDialog.tsx

```diff
@@ -38,18 +42,15 @@ const DeleteConversationDialog: React.FC<DeleteConversationDialogProps> = ({ onD
   return (
     <AlertDialog open={isAlertOpen} onOpenChange={setIsAlertOpen}>
       <AlertDialogTrigger asChild>
-        <button
+        <Trash
+          variant="Bold"
+          className={`${colorVariant === 'default' ? 'text-primary/40' : 'text-tertiary'} cursor-pointer transition-colors duration-200 hover:text-secondary`}
```

Comment:
no destructive color on hover?

Reply:
Hmm removed this because it didn't seem to actually do anything. Didn't spend too much time researching but doesn't seem like it's a valid tailwind class. Very curious what it's supposed to do though :)